blt/blt.yml

```diff
@@ -62,7 +62,7 @@ cm:
   allow-overrides: true
 sync:
   commands:
-    - 'custom:hello'
+    - 'blt:init:settings'
```

Comment:
You want to int the settings on every sync? Why?

Reply:
`custom:hello` no longer exists and the sync command actually runs `blt:init:settings` as the 2nd step. so this is just doing it twice. I do this because i dont like how blt tries to run composer in step 1